website/docs/r/access_group.html.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: "cloudflare"
+page_title: "Cloudflare: cloudflare_access_group"
+sidebar_current: "docs-cloudflare-resource-access-group"
+description: |-
+  Provides a Cloudflare Access Group resource.
+---
+
+# cloudflare_access_group
+
+Provides a Cloudflare Access Group resource. Access Groups are used
+in conjunction with Access Plolicies to restrict access to a
+particular resource.
+
+## Example Usage
+
+```hcl
+# Allowing access to `test@example.com` email address only
```

Comment:
We should also look to have an example where an Access Policy is referencing an Access Group.

Reply:
I don't think the Access Policy accepts Access Group at that time.

Reply:
You're right. The schema hasn't yet been updated. Thanks, I'll add a card to get it sorted first.